crates/figma_import/src/figma_schema.rs

```diff
@@ -1070,3 +1080,164 @@ pub struct ProjectFilesResponse {
     pub name: String,
     pub files: Vec<HashMap<String, String>>,
 }
+
+#[derive(Deserialize, Serialize, Debug, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct ModeBinding {
+    pub mode_id: String,
+    pub name: String,
+}
+
+#[derive(Deserialize, Serialize, Debug, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct VariableCollection {
+    pub default_mode_id: String,
+    pub id: String,
+    pub name: String,
+    pub remote: bool,
+    pub modes: Vec<ModeBinding>,
+    pub key: String,
+}
+
+#[derive(Deserialize, Serialize, Debug, Clone)]
+#[serde(rename_all = "SCREAMING_SNAKE_CASE")]
+pub enum VariableType {
+    Boolean,
+    Float,
+    String,
+    Color,
+}
+
+#[derive(Deserialize, Serialize, Debug, Clone, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct VariableAlias {
+    pub r#type: String,
```

Comment:
What does the "r#type" do in rust?

Reply:
"type" is a reserved keyword so using "r#type" lets you get around that.